src/cascade/saver/db/save_model_results_to_db.py

```diff
@@ -90,9 +90,12 @@ def _normalize_draws_df(draws_df, execution_context):
     node_to_location = {r.node_id: r.c_location_id for _, r in node_table.iterrows()}
 
     draws["location_id"] = draws.node_id.apply(lambda nid: node_to_location[nid])
-
-    draws["sex_id"] = draws.x_sex.apply(lambda x: {-0.5: 2, 0.5: 1}[x])
-    return draws.drop(["node_id", "weight_id", "x_sex"], "columns")
+    covariate_table = execution_context.dismodfile.covariate
+    sex_index = int(covariate_table[covariate_table.covariate_name == "sex"].covariate_id)
+    draws["sex_id"] = draws[f"x_{sex_index}"].apply(lambda x: {-0.5: 2, 0.5: 1}[x])
+    # Remove covariates from draws to upload.
+    to_drop = ["node_id", "weight_id"] + [str(cov_col) for cov_col in draws.columns if cov_col.startswith("x_")]
```

Comment:
Does it make more sense to white list columns?

Reply:
It was a blacklist. I'm not sure why. Need to have a longer conversation with Natalie. This is why we try to make the first choice correct, because the first choice lives a long time.